xdsl/dialects/pdl.py

```diff
@@ -721,6 +680,11 @@ class ReplaceOp(IRDLOperation):
 
     irdl_options = [AttrSizedOperandSegments()]
 
+    assembly_format = (
+        "$op_value `with` ` ` "
+        "(`(` $repl_values^ `:` type($repl_values) `)`)? ($repl_operation^)? attr-dict"
```

Comment:
Don't believe `repl_operation` needs to be in an optional group but understand if you want to keep this the same as MLIR

Reply:
Oh yeah, I really just copy pasted it so I didn't look at it properly. I'll change it, it doesn't *need* to be exactly the same